src/main/java/org/jabref/gui/JabRefGUI.java

```diff
@@ -68,6 +68,11 @@ private void openWindow(Stage mainStage) {
         mainFrame.init();
 
         GuiPreferences guiPreferences = preferencesService.getGuiPreferences();
+
+        // Set the min-width and min-height for the main window
```

Comment:
I don't think the comment is necessary

Reply:
@HoussemNasri Thank you for your feedback, I'll remove it if necessary.